rpc/grpc/grpc.go

```diff
@@ -86,6 +86,11 @@ func (g *GrpcEndpoint) Allow(ctx context.Context, req *pb.AllowRequest) (*pb.All
 			logging.Printf("Caught error %v", err)
 			rsp.Status = pb.AllowResponse_REJECTED_SERVER_ERROR
 		}
+
+		// Fail open
+		if rsp.Status == pb.AllowResponse_REJECTED_SERVER_ERROR {
+			rsp.Status = pb.AllowResponse_OK
```

Comment:
If we are returning a successful response, do we need to set `rsp.TokensGranted` and `rsp.WaitMillis` like below?

Reply:
We absolutely should be, added it.